### PR TITLE
Fix table alignment

### PR DESCRIPTION
**Pull request by JAremko**
Solve #2284 *"tables not aligned correctly in documentation view mode"*

#### After

*Also I removed boxes from the key codes  because they introduce vertical misalignment - kills vertical rhythm(harder to read) and horizontal misalignment - breaks tables.*

### CODE REVIEW

**Comment by JAremko** [comment]
@syl20bnr  :shipit:

**Comment by JAremko** [comment]
@syl20bnr  	*Wider keyboard-emphasized-regions d331469*
#### Now it's much better   


I think it's good enough  - I'm done :shipit:

**Comment by JAremko** [comment]
Ok. This time I'm really really done :smile:

**Comment by JAremko** [comment]
*Make sure that org-kbd-marker' if valid. eeda697*  - **In case if markers will change (paranoia mod)**

**Comment by JAremko** [comment]
I think I'll squash it. Too many commit for a single issue.

**Comment by JAremko** [comment]
@syl20bnr  :shipit:

**Comment by syl20bnr** [comment]
Seems to not work with the default theme :-(

capture d ecran 2016-04-15 a 22 53 41

**Comment by JAremko** [comment]
@syl20bnr The pony needs more tricks :confused:

**Comment by JAremko** [comment]
I'll use more precise color calculation method for space-doc mode as a whole.

**Comment by JAremko** [comment]
Each time I'm almost finished with this one... I come up with a better solution.
 I have rewritten the whole space-doc-mode like 3 times already.   Hope next one will be perfect :smile: 

**Just an update. I'm not going to give up :smiling_imp:**

**Comment by JAremko** [comment]
This one's almost drove me crazy. Couple of improvements here and there and it  is ready!

**Comment by JAremko** [comment]
Ok. I think this is it. I'll double check the code couple more tens of times. Work on doc strings and names.  And if it all good move to the Spacemacs core.

File: layers/+spacemacs/spacemacs-org/local/space-doc/space-doc.el [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+(defconst spacemacs--space-doc-modificators
+  '(spacemacs//space-doc-set-space-doc-cache
+    spacemacs//space-doc-modf-emphasis-overlays
+    spacemacs//space-doc-modf-meta-tags-overlays
+    spacemacs//space-doc-modf-link-protocol
+    spacemacs//space-doc-modf-org-block-line-face-remap
+    spacemacs//space-doc-modf-org-kbd-face-remap
+    spacemacs//space-doc-modf-advice-org-do-emphasis-faces
+    (lambda (flag) (spacemacs//space-doc-run-modfs-deferred
+               '()
```
Comment (JAremko) [inline layers/+spacemacs/spacemacs-org/local/space-doc/space-doc.el]: Might add something like "zebra tables" here one day.

**Comment by JAremko** [comment]
It seems that I can't break it anymore...

**Comment by JAremko** [comment]
@syl20bnr  :shipit:

**Comment by syl20bnr** [comment]
Very nice, I love the modern/flat/2016 top new style 👍 
Thank you !
Cherry-picked into develop branch, you can safely delete your branch.

**Comment by JAremko** [comment]
@syl20bnr  Thanks! Hope it won't blowup :wink:

**Pull request closed without merging**